### DISCUSSION TITLE
osu!editor sliders and audio

### DISCUSSION THREAD

**Discussion by CrashyWashy** [ppy/osu#27100] [Ideas] [answered]
Either it is incredibly unclear how to do it, or it is yet to be implemented, but slider ends or "tails" have no option for changing volume. This is strange to me, given that slider ends from osu!stable maps are still muted if chosen so by the mapper, yet there is no indication in the actual editor.

**Comment by bdach** [comment, marked as answer]
basically the same as https://github.com/ppy/osu/issues/22187
